the-collab-lab/tcl-34-smart-shopping-list
# [Bug] Time frames radio group

_For an example of how to fill this template out, [see this Pull Request](https://github.com/the-collab-lab/tcl-3-smart-shopping-list/pull/44)._ 

## Description

Bug 🐞:
It happens when you want to create a new product and you have already selected one of the purchase frequency options (soon, not-soon, kind-of-soon), the radio button is "reset" or always returns to the first radio button.

This PR fixes the timeframe radio group bug, removes the unused component `RadioInput` and adds accessibility tags to menu buttons. The problem was with the custom radio button component. When interacting with the form the component got lost in the formState references. The solution was to use the native radio buttons.

## Related Issue

Closes #26 

## Type of Changes



|     | Type                       |
| --- | -------------------------- |
|  ✓ | :bug: Bug fix              |
|    | :sparkles: New feature     |
|  ✓ | :hammer: Refactoring       |
|    | :100: Add tests            |
|    | :link: Update dependencies |
|    | :scroll: Docs              |

## Updates

### Before

![incorrect](https://user-images.githubusercontent.com/32148199/143053098-e1acdfd7-00f4-49d8-aea6-d62f9231fdb4.gif)

### After

![correct](https://user-images.githubusercontent.com/32148199/143053122-65db7880-7b31-46b5-b7ec-8c6e59ace2be.gif)

## Testing Steps / QA Criteria

- From your terminal, pull down this branch with `git pull time-frame-bu` and check that branch out with `git checkout time-frame-bug`
- npm start to launch the app
- Fill in the "Shared token" field with the `isa-collection` token
- Access the add item screen and try to create a new product
